sources/packages/backend/libs/integrations/src/services/disbursement-schedule-service/disbursement-schedule-service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from "@nestjs/common";
+import { RecordDataModelService, DisbursementSchedule } from "@sims/sims-db";
+
+/**
+ * Service layer for Student Application disbursement schedules.
+ */
+@Injectable()
+export class IntegrationDisbursementSchedulerService extends RecordDataModelService<DisbursementSchedule> {
```

Comment:
I am ok to have the suffix `Integration` if this service is not supposed to be consumed anywhere but the integrations.
I can see this one associated with `API` and `queue-consumers`, is it not the case to have it at `@sims/services` level?